src/rtctools_heat_network/pycml/component_library/heat/source.py

```diff
@@ -6,6 +6,16 @@
 
 
 class Source(_NonStorageComponent):
+    """
+    The source component is there to insert thermal power (Heat) out of the system.
```

Comment:
If think the wrong word was used: " ... to insert thermal power (Heat) ?into? the system ...". System being the network?

Reply:
done